src/ansys/aedt/core/modules/boundary.py

```diff
@@ -245,6 +246,45 @@ def __init__(self, app, component_type, component_name, props):
         self.native_properties = self.props["NativeComponentDefinitionProvider"]
         self.auto_update = True
 
+    @property
+    def name(self):
+        """Name of the object.
+
+        Returns
+        -------
+        str
+           Name of the object.
+
+        """
+        return self._name
+
+    @name.setter
+    def name(self, component_name):
+        if component_name not in self._app.native_component_names:
+            if component_name != self._name:
```

Comment:
```suggestion
        if component_name != self._name:
            if component_name not in self._app.native_component_names:
```
This makes me think about #5100 (which I should handle soon). I would recommend switching the order as mentioned by @adamgregory81.